deploy/helm/charts/templates/cluster-exporter.yaml

```diff
@@ -16,9 +16,9 @@ spec:
     metadata:
       labels:
         {{- include "openebs-ndm.cluster-exporter.labels" . | nindent 8 }}
-          {{- with .Values.ndmExporter.clusterExporter.podLabels }}
-          {{ toYaml . | nindent 8 }}
-          {{- end }}
+        {{- with .Values.ndmExporter.clusterExporter.podLabels }}
+        {{ toYaml . | nindent 8 }}
+        {{- end }}
```

Comment:
Is it fine to remove the indentation here?

Reply:
Yup. Tested it. Works fine. The indentation actually was not required because we are specifying it explicitly.

Reply:
Ok